awx/main/migrations/0128_unifiedjob_installed_collections.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.16 on 2021-02-16 20:27
+
+import awx.main.fields
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0127_reset_pod_spec_override'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='unifiedjob',
+            name='installed_collections',
+            field=awx.main.fields.JSONBField(blank=True, default=dict, editable=False, help_text='The Collections names and versions installed in the execution environment.'),
```

Comment:
@wenottingham @shanemcd @beeankha should we go ahead and update the analytics code to include this new data in what we ship to Analytics?

https://github.com/ansible/awx/blob/devel/awx/main/analytics/collectors.py#L316-L338

Reply:
Yes, although cc @Ladas for updating the processor.

Reply:
Yep, I agree that we should go ahead and do this here.